testdata/networking/multus-cni/Pods/IPv6/ipv6-pod.yaml

```diff
@@ -0,0 +1,25 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: test-pod
+spec:
+  replicas: 3
+  selector:
+    matchLabels:
+      name: test-pod
+  template:
+    metadata:
+      annotations:
+        k8s.v1.cni.cncf.io/networks: whereabouts-excludeipv6
+      labels:
+        name: test-pod
+    spec:
+      containers:
+      - name: test-pod
+        image: quay.io/openshifttest/hello-sdn@sha256:d5785550cf77b7932b090fcd1a2625472912fb3189d5973f177a5a2c347a1f95
```

Comment:
Is it standard and required to hardcode image sha256 tag? Or we can just leave `quay.io/openshifttest/hello-sdn` and let it fetch the latest image? Ideally image should never change in such a way that it will break any existing tests right?

Reply:
The image name is fixed.

Reply:
Is that standard way we do things in this repo?

Reply:
The way we did here "image: quay.io/openshifttest/hello-sdn@sha256:d5785550cf77b7932b090fcd1a2625472912fb3189d5973f177a5a2c347a1f95" is standard way. sha256:d5785550cf77b7932b090fcd1a2625472912fb3189d5973f177a5a2c347a1f95 is container image ID, different image has different ID. For this testing I use this container image

Reply:
I agree its not a compulsion now. We use https://quay.io/organization/openshifttest as our qe testfiles inventory repo. In past I had seen it to use different tags for whatever reason so we got accustomed to this sha256 practise. But seems like its okay to point to latest now without specifying tag.

Reply:
@anuragthehatter thanks for clarifying. I'd like to see if we could stick to latest, in my limited understanding, it seems best to do so. That way all our tests running same image, not different SHAs. And that way if a test fails, we know its not due to an image. And if we see many tests fail, we know we broke the image.

Reply:
@kedark3 Just remove sha256 in the new commit. Thanks!

Reply:
Oh wait. digest is indeed required if we are testing on disconnected cluster as per #850. Disconnected clusters (not connected to internet or via proxies) will setup registry locally on the cluster itself and need digests to access the image cc @zhaozhanqi WDYT

Reply:
yes, keeping the digest is the recommended way to go...not very human readable though.

Reply:
oh interesting. I wish there was way to just say `latest` and not worry.

Reply:
digest is indeed required for disconnected. I remembered tag is not be supported before. So better use digest

Reply:
@zhaozhanqi Thanks for the confirmation. The new commit add digest back to image name for disconnected cluster.

Reply:
@weliang1 sorry if I caused confusion here. Thanks for putting up with changes twice. I know it can be unpleasant..

Reply:
@kedark3 No worry, it's process to learn something new.